src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/Conversions.cs

```diff
@@ -308,18 +309,19 @@ private static ConversionKind ClassifyImplicitConstantExpressionConversion(Bound
                 if (nt.OriginalDefinition.GetSpecialTypeSafe() == SpecialType.System_Nullable_T &&
                     HasImplicitConstantExpressionConversion(source, nt.TypeArgumentsNoUseSiteDiagnostics[0]))
                 {
-                    return ConversionKind.ImplicitNullable;
+                    return new Conversion(ConversionKind.ImplicitNullable, Conversion.ImplicitConstantUnderlying);
                 }
             }
 
-            return ConversionKind.NoConversion;
+            return Conversion.NoConversion;
         }
 
-        private ConversionKind ClassifyImplicitTupleLiteralConversion(BoundExpression source, TypeSymbol destination, ref HashSet<DiagnosticInfo> useSiteDiagnostics)
+        private Conversion ClassifyImplicitTupleLiteralConversion(BoundTupleLiteral source, TypeSymbol destination, ref HashSet<DiagnosticInfo> useSiteDiagnostics)
```

Comment:
Consider renaming "ClassifyImplicitTupleLiteralConversion" to "GetImplicitTupleLiteralConversion", for consistency with "GetImplicitTupleLiteralConversion".